### PR TITLE
f_tes_bash_script_get_tes-adapter-jar

### PR DESCRIPTION
**Pull request by evgeniimv**
added bash script to get tes-adapter .jar file from git-repo

### CODE REVIEW

File: tes-adapter/scripts/run_get_tes-adapter_jar.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+else
+    gitRepoUrl=$1
+fi
+
+mkdir tmp;
+cd tmp;
+
+tmpdir=$(pwd);
+
+git clone $gitRepoUrl $tmpdir;
```
Comment (SilinPavel) [inline tes-adapter/scripts/run_get_tes-adapter_jar.sh]: what if something goes wrong? lets implement check of status of the command and log to user that something wrong

File: tes-adapter/scripts/run_get_tes-adapter_jar.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+./gradlew bootJar;
```
Comment (SilinPavel) [inline tes-adapter/scripts/run_get_tes-adapter_jar.sh]: the same here

File: tes-adapter/scripts/run_get_tes-adapter_jar.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+echo "Copying jar-file from tes-adapter to $workdir"
```
Comment (SilinPavel) [inline tes-adapter/scripts/run_get_tes-adapter_jar.sh]: also log the other stage of the script like cloning, building

File: tes-adapter/scripts/run_get_tes-adapter_jar.sh (hunk 2 above)
Comment (SilinPavel) [inline tes-adapter/scripts/run_get_tes-adapter_jar.sh]: in the automation script we do not need to see all output that is generated by tools, lets hide it and simply notify user that we start some operation and done it, or fail if something wrong

File: tes-adapter/scripts/run_get_tes-adapter_jar.sh (hunk 1 above)
Comment (evgeniimv) [reply tes-adapter/scripts/run_get_tes-adapter_jar.sh]: fixed

File: tes-adapter/scripts/run_get_tes-adapter_jar.sh (hunk 2 above)
Comment (evgeniimv) [reply tes-adapter/scripts/run_get_tes-adapter_jar.sh]: fixed

File: tes-adapter/scripts/run_get_tes-adapter_jar.sh (hunk 3 above)
Comment (evgeniimv) [reply tes-adapter/scripts/run_get_tes-adapter_jar.sh]: added

File: tes-adapter/scripts/run_get_tes-adapter_jar.sh (hunk 2 above)
Comment (evgeniimv) [reply tes-adapter/scripts/run_get_tes-adapter_jar.sh]: fixed

File: tes-adapter/scripts/run_get_tes-adapter_jar.sh [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+cd tmp;
+tmpdir=$(pwd);
+
+#Step 1
+echo "Cloning git-repo from $gitRepoUrl"
+git clone $gitRepoUrl $tmpdir || { echo "Cloning failed!" >&2; exit 1; };
+
+#Step 2
+echo "Building jar-file..."
+./gradlew bootJar &> /dev/null || { echo "Building jar-file failed!" >&2; exit 1; };
```
Comment (SilinPavel) [inline tes-adapter/scripts/run_get_tes-adapter_jar.sh]: lets redirect only stdout but stderr leave as is

Comment (evgeniimv) [reply tes-adapter/scripts/run_get_tes-adapter_jar.sh]: Unfortunately, it looks like separately "> /dev/null"  or  "2> /dev/null" do not solve the problem. The output is still presents in terminal. Seems the only possible way to hide output is use:
1)  &> /dev/null
2) > /dev/null 2>&1
3) > /dev/null 2> /dev/null

**Review by SilinPavel — APPROVED** [review]

**Pull request merged**